crates/era-cheatcodes/src/cheatcodes.rs

```diff
@@ -1469,6 +1469,22 @@ impl CheatcodeTracer {
                     tracing::error!("👷 Setting nonces failed")
                 }
             }
+            sign_0(sign_0Call { privateKey: private_key, digest }) => {
+                tracing::info!("👷 Signing digest with private key");
+                let Ok(signature) = zksync_types::PackedEthSignature::sign(
+                    &private_key.to_h256(),
+                    digest.as_slice(),
+                ) else {
+                    tracing::error!("Failed to sign digest with private key");
+                    return
+                };
+
+                let r = signature.r();
+                let s = signature.s();
+                let v = signature.v() + 27;
```

Comment:
Why `+ 27`?

Reply:
Is because of the way PackedEthSignature work. 
```
/// Ethereum signed message produced by most clients contains v where v = 27 + recovery_id(0,1,2,3),
/// but for some clients v = recovery_id(0,1,2,3). (this is the case here)
```

Reply:
Maybe we can add a comment or link to this doc

Reply:
Will do